server/log_rpc_server.go

```diff
@@ -745,3 +761,13 @@ func (t *TrillianLogRPCServer) InitLog(ctx context.Context, req *trillian.InitLo
 func spanFor(ctx context.Context, name string) (context.Context, *trace.Span) {
 	return trace.StartSpan(ctx, fmt.Sprintf("%s.%s", traceSpanRoot, name))
 }
+
+// percentileBuckets returns a range of buckets for 0.0-100.0% use cases.
+// in 1.0% increments.
+func percentileBuckets() []float64 {
```

Comment:
Maybe make this `monitoring.PercentileBuckets(increment int)` (and have it live in the add-new-monitoring-entrypoint commit).

Reply:
Not quite clear if you're suggesting to pull this out of the prometheus related code or just out of that commit?

Reply:
Sorry that might be confusing due to the way I moved code around. I'll try again. do you think we should have a create method for each histogram type, which seemed odd to me or make the bucket creation public and stick to one new one?

Reply:
I was just thinking if that some other part of the code wanted a metric with specified percentile buckets it might be nice if the helper to create those buckets (ready for feeding into `NewHistogramWithBuckets`) was generally available.   No big deal though -- we could move it across at the point where we need another one.

Reply:
OK I'll go and have a think.